src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -14,7 +14,9 @@
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <XPLATProject>true</XPLATProject>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);CreatePackNupkg</TargetsForTfmSpecificContentInPackage>
+    <TargetsForTfmSpecificContentInPackage Condition=" '$(Create2ndNupkg)' == 'false'">$(TargetsForTfmSpecificContentInPackage);CreatePackNupkg</TargetsForTfmSpecificContentInPackage>
```

Comment:
In MSBuild you don't want to test for false. 
The property is a string, not a boolean. You can't guarantee that someone will set it to false exactly. 

The correct thing to do here is `$(Create2ndNupkg) != 'true'`

Reply:
Thanks! Changed accordingly.